src/Viewer.js

```diff
@@ -614,7 +646,7 @@ Viewer.prototype.switchScene = function(newScene, opts, done) {
   // Remove old layer when tween is complete
```

Comment:
s/layer/layers/

Reply:
what exactly do you mean?

Reply:
Replace "layer" with "layers" in the comment.